### PR TITLE
Fix ivar odds and ends

### PR DESCRIPTION
**Pull request by wm75**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

- Circumvents a bug in ivar getmasked that prevented the tool
from masking all primers of an amplicon.
- Unifies the getmasked and removereads tools in one wrapper
- Adds the possibility to specify more than two primers per amplicon in
  the getmasked primer "pair" file
- Makes ivar trim and ivar getmasked/removereads accept non-standard,
  but widely used BED files with non-integer values in the score column
- Makes ivar trim and ivar removereads reuse BAM index files

### CODE REVIEW

File: tools/ivar/completemask.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        for primer in masked_primers:
+            for amplicon in amplicon_data:
+                if primer in amplicon:
+                    masked_complete += amplicon
+        result = '\t'.join(sorted(set(masked_complete)))
+        print()
+        print('Removing reads primed with any of:')
+        print(result)
+        with open(sys.argv[1], 'w') as o:
+            o.write(result)
```
Comment (bgruening) [inline tools/ivar/completemask.py:26]: Can we put this into one statement please

File: tools/ivar/ivar_removereads.xml [hunk 2]
```diff
@@ -1,45 +1,74 @@
...
         </test>
     </tests>
     <help><![CDATA[
-        This command accepts an aligned and sorted BAM file trimmed using ivar trim
-        and removes the reads corresponding to the supplied primer indices, which is
-        the output of ivar getmasked command. Under the hood, ivar trim adds the
-        zero based primer index (based on the BED file) to the BAM auxillary data for
-        every read. Hence, ivar removereads will only work on BAM files that have
-        been trimmed using ivar trim.
+This is a Galaxy wrapper combining the functionality of ``ivar getmasked`` and
```
Comment (bgruening) [inline tools/ivar/ivar_removereads.xml:53]: This tool combines ...?

**Comment by wm75** [comment]
@bgruening What about the now abandoned `ivar getmasked` tool. Should I remove it?

**Comment by bgruening** [comment]
Yes, remove it. We need to deprecate it in the TS then.

**Comment by bgruening** [comment]
@thanhleviet can you have a look at this one?

File: tools/ivar/completemask.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import sys
```
Comment (thanhleviet) [inline tools/ivar/completemask.py:1]: Could you add shebang line ?

File: tools/ivar/sanitize_bed.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import sys
```
Comment (thanhleviet) [inline tools/ivar/sanitize_bed.py:1]: shebang line for python

**Pull request merged**